addons/mail/static/src/new/core/thread_model.js

```diff
@@ -111,4 +113,34 @@ export class Thread {
     sortMessages() {
         this.messages.sort((msgId1, msgId2) => msgId1 - msgId2);
     }
+
+    get onlineMembers() {
+        const orderedOnlineMembers = [];
+        for (const member of this.channelMembers) {
+            const memberImStatus = this._state.partners[member.partnerId].im_status;
+            member.im_status = memberImStatus;
+            if (member.partnerId !== this._state.user.partnerId) {
+                member.hasOpenChatFromAvatarClick = true;
+            }
+            if (memberImStatus === "online") {
+                orderedOnlineMembers.push(member);
+            }
+        }
+        return orderedOnlineMembers.sort((p1, p2) => (p1.name < p2.name ? -1 : 1));
```

Comment:
Can be a follow-up PR: would be nice to introduce `channel_member_model`, so that
```js
this._state.partners[member.partnerId].im_status;
```
becomes simply
```js
member.partner.im_status
```